progress-reports/icon-bridge/ICON_Bridge-Near_Integration-November-22-progress-report.md

```diff
@@ -0,0 +1,29 @@
+# Progress Report
+
+**Project:**  ICON Bridge - Near Integration  
+**Proposal:** Private  
+**Team:** HugoByte  
+**Reporting Period:** 01 Nov 2022 - 30 Nov 2022
+
+## Summary
+
+As per the new direction, the focus has been shifted to enabling IBC for all the integrations which will further widen the interoperability of ICON Hub in the Cosmos and Interchain ecosystem
```

Comment:
This progress report should include the work done on the ICON Bridge. There were several PRs done over the past month. Please include them.

Reply:
I have updated the report. kindly review.